### ISSUE TITLE
TYPO3 9.x support

### ISSUE THREAD

**Issue by fnagel** [fnagel/beautyofcode#25]
See https://github.com/fnagel/beautyofcode/pull/23 and https://github.com/fnagel/beautyofcode/tree/typo3-9

**Comment by fnagel** [comment]
Hey @garbast, hey @sebkln,

I've changed the PHP namespace and package name, finished testing and fixed all Functional and Unit tests. Would you mind to test the latest `typo3-9` branch before TER release? Some final feedback if everything works as expected would be great!

**Comment by sebkln** [comment]
Hey, thank you for your work. I will test the latest version by the end of the week.

**Comment by sebkln** [comment]
Now had time to test:

- [x] The former error message (_An error occurred trying to process items for field "Programming Language" […]_) when opening a BOC content element is gone now.
- [x] The select list of programming languages is taken correctly from my custom TypoScript constants.
- [x] The t3editor uses the correct syntax highlighting for the chosen (and supported) language.
- [x] No changes in frontend: gutter and/or line highlighting is correctly applied.

In short, I didn't notice any issues with the `typo3-9` branch.

Again, thank you for your ongoing efforts to support this extension in TYPO3 v9!

**Comment by fnagel** [comment]
@sebkln Thanks for reviewing!

Version 4.0.0 has just been released to TER!
